### ISSUE TITLE
[feature] Implement bandwith limiter 

### ISSUE THREAD

**Issue by vevarm** [NebulousLabs/Sia#1554]
Now when sia got a speed boost I may be useful to have bw limiter. Not prio but sometime in future.

**Comment by DavidVorick** [comment]
I don't think it will be too difficult to add, we may be able to get it into v1.1.1 or v1.1.2.

**Comment by tbenz9** [comment]
Recommendation to devs: label "Feature Request"
